java/ql/lib/semmle/code/java/PrettyPrintAst.qll

```diff
@@ -743,10 +746,14 @@ private class PpSwitchStmt extends PpAst, SwitchStmt {
 }
 
 private class PpSwitchCase extends PpAst, SwitchCase {
+  PpSwitchCase() { not this instanceof PatternCase }
+
   override string getPart(int i) {
     i = 0 and result = "default" and this instanceof DefaultCase
     or
-    i = 0 and result = "case " and this instanceof ConstCase
+    i = 0 and result = "case " and not this instanceof DefaultCase
```

Comment:
This needs to be updated now that `NullDefaultCase` is included in `DefaultCase`.

Reply:
I guess we ought to have a qltest for this library...